WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -40346,19 +40368,38 @@ e_latin_empire = {
 	capital = 496 # Byzantion
 	
 	short_name = yes
+	dynasty_title_names = no
 	
 	
 	allow = {
 		OR = {
+			has_romance_culture_trigger = yes
+			culture_group = latin
 			religion = catholic
 			is_heresy_of = catholic
 		}
-		e_byzantium = {
-			has_holder = no
+
+		NOT = {
+			culture = greek
+			culture = roman
+			religion = orthodox
+			religion = iconoclast
 		}
+
+		NOT = { has_landed_title = e_byzantium }
+
 		e_roman_empire = {
 			has_holder = no
 		}
+		
+		496 = { # Constantinople
+			owner = {
+				OR = {
+					is_liege_or_above = ROOT
+					character = ROOT
+				}
+		}
+
```

Comment:
First of all this condition isn't necessary, secondly remember that this is the Latin Empire.